### PR TITLE
Sam causal

### PR DESCRIPTION
**Pull request by sdbpimentel**

### CODE REVIEW

File: 5-causation/01-defining-causality/notes.qmd [hunk 1]
```diff
@@ -291,10 +295,6 @@ we need to think about how the data for the two groups was generated and
 whether they serve as good approximations to each other's counterfactuals.
 This question will be the focus of the rest of the causality unit.
 
-
-:::
```
Comment (andrewpbray) [inline 5-causation/01-defining-causality/notes.qmd:295]: These ::: open and close pandoc cells in a similar way to ``` does for code cells. It looks like this remove messed up the formatting a bit so I am going to put it back in.

File: 5-causation/01-defining-causality/notes.qmd [hunk 2]
```diff
@@ -283,17 +283,9 @@ df %>%
...
-because we see subjects under both treatment and control conditions.
-But to decide if estimates like the one we've just given are reliable
-we need to think about how the data for the two groups was generated and
-whether they serve as good approximations to each other's counterfactuals.
-This question will be the focus of the rest of the causality unit.
-
-
 :::
 
+It is easier to come up with plausible-seeming estimates of average treatment effects than individual treatment effects because we see subjects under both treatment and control conditions. But to decide if estimates like the one we've just given are reliable we need to think about how the data for the two groups was generated and whether they serve as good approximations to each other's counterfactuals. This question will be the focus of the rest of the causality unit.
```
Comment (andrewpbray) [inline 5-causation/01-defining-causality/notes.qmd:288]: I also fixed the paragraph formatting here (better to not do manual line breaks on paragraphs)

File: 5-causation/01-defining-causality/slides.qmd [hunk 3]
```diff
@@ -26,6 +26,7 @@ execute:
 
 -  Announcements
 -  Concept Questions
+-  Course Evaluations
```
Comment (andrewpbray) [inline 5-causation/01-defining-causality/slides.qmd]: Was the verdict to reserve class time for this? It sounded to me like Jeremy thought we should nix this give the extra CQ.

**Review by andrewpbray — APPROVED** [review]
Looks good! I appreciate the attention to detail in these materials - really getting them refined so that we can work with them for semesters to come!

File: 5-causation/01-defining-causality/slides.qmd (hunk 3 above)
Comment (sdbpimentel) [reply 5-causation/01-defining-causality/slides.qmd]: Good catch, I added this at the beginning of the instructor meeting but meant to remove it.  Just re-pushed with this removed.

**Pull request merged**